### PR TITLE
Add a job to test ci with build01

### PR DESCRIPTION
**Pull request by hongkailiu**
/cc @openshift/openshift-team-developer-productivity-test-platform

### CODE REVIEW

File: ci-operator/jobs/openshift/ci-secret-mirroring-controller/openshift-ci-secret-mirroring-controller-master-presubmits.yaml [hunk 1]
```diff
@@ -48,6 +48,54 @@ presubmits:
...
+            cpu: 10m
+        volumeMounts:
+        - mountPath: /etc/sentry-dsn
+          name: sentry-dsn
+          readOnly: true
+      serviceAccountName: ci-operator
+      volumes:
+      - name: sentry-dsn
+        secret:
+          secretName: sentry-dsn
```
Comment (hongkailiu) [inline ci-operator/jobs/openshift/ci-secret-mirroring-controller/openshift-ci-secret-mirroring-controller-master-presubmits.yaml]: /hold

@stevekuznetsov 
How do we do mirroring this secret `sentry-dsn`?

Comment (hongkailiu) [reply ci-operator/jobs/openshift/ci-secret-mirroring-controller/openshift-ci-secret-mirroring-controller-master-presubmits.yaml]: https://jira.coreos.com/browse/DPTP-639

**Comment by hongkailiu** [comment]
@stevekuznetsov 
rehearsal passed.

Created the cards for the missing items. The cards are listed: https://jira.coreos.com/browse/DPTP-612

**Comment by stevekuznetsov** [comment]
This is so exciting!!!

**Comment by hongkailiu** [comment]
This means: plank is working for create prowjobs and deck is working for the pod logs
https://prow.svc.ci.openshift.org/view/gcs/origin-ci-test/pr-logs/pull/openshift_release/5650/rehearse-5650-pull-ci-openshift-ci-secret-mirroring-controller-master-format-build01/812

Let us trust for now sinker can delete the prowjobs/pods.

In order to use `ci-operator:lastest` image, we need to give the full path (it is now protected by auth). We need to set up pull-secret properly. This is more essential for pushing the built images.

```
  Warning  Failed     2m55s (x2 over 3m10s)  kubelet, ip-10-0-141-110.ec2.internal  Failed to pull image "registry.svc.ci.openshift.org/openshift": rpc error: code = Unknown desc = unable to retrieve auth token: invalid username/password
  Warning  Failed     2m42s (x3 over 3m9s)   kubelet, ip-10-0-141-110.ec2.internal  Error: ImagePullBackOff
```

This item is described in https://jira.coreos.com/browse/DPTP-641


We are ready for merge.

**Pull request merged**